src/taipy/core/_entity/_migrate.py

```diff
@@ -23,9 +24,6 @@
 
 
 def __update_parent_ids(entity: Dict, data: Dict) -> Dict:
-    # parent_id was replaced by parent_ids
-    _ = entity.pop("parent_id", None)
```

Comment:
Not sure I understand why we are removing this line.

Reply:
Because we will pop it when we handle the `parent_id` -> `owner_id` migration.

`parent_id` and `parent_ids` are actually not related so I don't prefer doing it here.